node/src/service.rs

```diff
@@ -491,7 +496,7 @@ where
 		keystore,
 		local_role: role,
 		telemetry: telemetry.as_ref().map(|x| x.handle()),
-		protocol_name: Default::default(),
+		protocol_name: grandpa_protocol_name,
```

Comment:
hmm, this doesn't sync at all now

```
2022-02-17 13:37:37 add_set_reserved_peer with unknown protocol: /0xd2a6…86fc/grandpa/1  
```

we are probably already using some protocol name

Reply:
ok, looks like fallback kicked in and it synces like before, but still logging same message

Reply:
Is it Worth effort to investigate further ?

Reply:
yes, even if its just visual it will be reported by out validators